bundle/Templating/Twig/Extension/NamedObjectRuntime.php

```diff
@@ -21,33 +23,66 @@ class NamedObjectRuntime
      */
     private $namedObjectProvider;
 
-    public function __construct(Provider $specialObjectProvider)
-    {
+    /**
+     * @var \Psr\Log\LoggerInterface
+     */
+    private $logger;
+
+    /**
+     * @var bool
+     */
+    private $isDebug;
+
+    public function __construct(
+        Provider $specialObjectProvider,
+        LoggerInterface $logger,
```

Comment:
Logger can be `null` if configured so, so here it should be `?LoggerInterface $logger` and below `$this->logger = $logger ?? new NullLogger();`

Reply:
Updated: https://github.com/netgen/ezplatform-site-api/pull/164/commits/0a7d31ee8fd53538aecd9263d0a2ce90f7da14e8